x/group/server/invariants.go

```diff
@@ -0,0 +1,96 @@
+package server
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/regen-network/regen-ledger/orm"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/regen-network/regen-ledger/types"
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+func (s serverImpl) RegisterInvariants(ir sdk.InvariantRegistry) {
+	ir.RegisterRoute(group.ModuleName, "Tally-Votes", s.TallyVotesInvariant())
+}
+
+func (s serverImpl) AllInvariants() sdk.Invariant {
+	return func(ctx sdk.Context) (string, bool) {
+		return s.TallyVotesInvariant()(ctx)
+	}
+}
+
+func (s serverImpl) TallyVotesInvariant() sdk.Invariant {
+	return func(sdkCtx sdk.Context) (string, bool) {
+		var msg string
+		var broken bool
+		var curProposal []*group.Proposal
+		ctx := types.Context{Context: sdkCtx}
+		it1, err := s.proposalTable.PrefixScan(ctx, 1, math.MaxUint64)
+		if err != nil {
+			return "start value must be less than end value in iterator", false
```

Comment:
I'm wondering if we shouldn't actually just panic instead in case of error? See for instance in cosmos-sdk, `x/distribution/keeper/invariants.go`

Reply:
Yeah, I'll do that